### ISSUE TITLE
Google Has Started Blocking SignIns since using K9 Mail

### ISSUE THREAD

**Issue by NovaViper** [k9mail/k-9#3661]
### Expected behavior
Should not be getting messages about blocked sign in attempts from Google

### Actual behavior
Ever since I signed into my secondary gmail accounts (which do not use 2 Step Authentication like my primary does), I give been getting emails from Google saying that multiple sign in attempts have been blocked by "someone using my password", and almost soon after I notice that K9 can no longer get into the accounts to see emails, I have to re-log in nearly every time. The emails look like this:
![image](https://user-images.githubusercontent.com/7191115/46900864-14c5d000-ce6f-11e8-8fd3-5c2145449837.png)

### Steps to reproduce
1. Sign into K9 Mail using a Gmail account that doesn't have 2-Step Authentication activated
2. Wait for message from Google in a couple of days
3. Notice that you can no longer access the account since that email

### Environment
K-9 Mail version: 5.600

Android version: 8.0.0

Account type (IMAP, POP3, WebDAV/Exchange): Gmail (IMAP)

**Comment by lieryan** [comment]
Did you enable [Less secure Apps](https://support.google.com/accounts/answer/6010255?hl=en) on your secondary account?

**Comment by NovaViper** [comment]
Yeah.. but it still keeps on sending me those notifications

**Comment by NovaViper** [comment]
@lieryan I just took a look at the issue.. it doesn't appear to have been implemented/solved at all.

**Comment by ByteHamster** [comment]
Have you tried changing your password? Maybe someone found out your password and it is just a coincidence that the messages started after installing K-9. K-9 is a quite widespread app and it does not look like many people have this issue.

**Comment by NovaViper** [comment]
No it's not the password only happens when  I have added the accounts to K9. I also seem to notice it happens with  (Did not have this with a my other client) my VPN

**Comment by RafaelLinux** [comment]
I have had the same message in my first try to access my gmail account from K9-mail. However, when I tried BlueMail, I have not that problem!!! Why K9 is not working out of the box with gmail accounts?

**Comment by wiktor-k** [comment]
Have you tried using Google's [app passwords](https://myaccount.google.com/apppasswords)? I recently set up K9 and didn't have problems with it. (tl;dr google generates you a second password that you can use in K9, username stays the same).

**Comment by lieryan** [comment]
@wiktor-k: App passwords only work if you enable Less secure apps. In some Google Apps for Business configuration, the administrator didn't allow you to enable Less secure apps, and so you can't create App passwords.

**Comment by wiktor-k** [comment]
Yep, app passwords are also disabled when [Advanced Protection](https://landing.google.com/advancedprotection/) is turned on.

I'm afraid only [XOAUTH2](https://github.com/k9mail/k-9/issues/655) could work in that case.

**Comment by NovaViper** [comment]
App passwords only seem to work when I have 2step authentication enabled

**Comment by leophys** [comment]
I have 2 steps auth and was using app specific password but since yesterday I am not able to login with k9 anymore on my gmail account. It complains that PLAIN authentication method is not allowed. Then it prompts an error message suggesting to follow a link with my browser (screenshot of the [error](https://ibb.co/02BCbF3) ) but I am not able to copy or select the text.

It happened something similar with conversations and google xmpp very recently.

**Comment by NovaViper** [comment]
Why hasn't support for XOAUTH2 been added to the app?
